language/Pattern-Matching.md

```diff
@@ -176,6 +176,30 @@ compare _ _ = EQ
 
 (The name `otherwise` is a synonym for `true` commonly used in guards.)
 
+Guards may also be used within `case` expressions, which allow for inline expressions. For example, these are equivalent: 
+
+```purs
+fb :: Int -> Effect Unit
+fb n =
+  log case n of
+    _
+      | 0 == mod n 15 -> "FizzBuzz"
+      | 0 == mod n 3 -> "Fizz"
+      | 0 == mod n 5 -> "Buzz"
+      | otherwise -> show n
+```
```

Comment:
I'm happy with the example. I think I most commonly see this done with a `case _ of` in the wild, as the variable name is only used once then and closer to the guards. What about this simplified version?

```purs
fizzBuzz :: Int -> String
fizzBuzz = case _ of
  n
    | 0 == mod n 15 -> "FizzBuzz"
    | 0 == mod n 3 -> "Fizz"
    | 0 == mod n 5 -> "Buzz
    | otherwise -> show n
```

or if you want to stay in `Effect` we can do `fizzBuzz = log <<< case _ of`